TestsUtils/CompileAndExtractTests.lean

```diff
@@ -196,10 +196,17 @@ end Transpilation
 
 section Ipld
 
+def findDiff [BEq α] (as bs : List α) : Option (Nat × α × α) :=
```

Comment:
I forgot to delete this one. Maybe it can be useful again?

Reply:
It may be worth keeping it in YatimaStdLib, but honestly I think it's a bit specialized so I'll just delete it for now.